definitions/variable/technology/electricity-operation.yaml

```
@@ -12,360 +12,84 @@
-    description: Active power of New+Imported biomass power plants biomass from dedicated
-      energy crops and biomass imports.
-    unit: MWh
-- Active Power|Electricity|Nuclear:
-    description: Active power of nuclear power plants
-    unit: MWh
-- Active Power|Electricity|Gas:
-    description: Active power of Gas power plants
-    unit: MWh
-- Active Power|Electricity|Gas|OCGT:
-    description: Active power of Open Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Gas|CCGT:
-    description: Active power of Combined Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Coal:
-    description: Active power of coal power plants
-    unit: MWh
-- Active Power|Electricity|Lignite:
-    description: Active power of Lignite power plants
-    unit: MWh
-- Active Power|Electricity|Oil:
-    description: Active power of oil power plants
-    unit: MWh
-- Active Power|Electricity|Geothermal:
-    description: Active power of Geothermal power plants
-    unit: MWh
-- Active Power|Electricity|Hydro|Reservoir:
-    description: Active power of "simple Hydro reservoir " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Pumped Storage:
-    description: Active power of "Pumped Storage " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Run of River:
-    description: Active power of "Run of River" units
-    unit: MWh
-- Active Power|Electricity|Hydro|Ocean:
-    description: Active power of Ocean power plants
-    unit: MWh
-- Active Power|Electricity|Solar:
-    description: Active power of Photovoltaic power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OnShore:
-    description: Active power of  Wind Onshore power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OffShore:
-    description: Active power of  Wind OffShore power plants
-    unit: MWh
+- Active Power|Electricity|<Fuel>:
+    description: Active power of <Fuel> power plants
+    unit:
+    - GWh
+
 - Active Power|Electricity|Load Curtailment:
     description: Energy curtailed - from the system point of view is equivalent to
       generation of Active Power
-    unit: MWh
+    unit:
+    - GWh
+
 - Active Power|Electricity|Load Shifting:
     description: Energy shifted - from the system point of view is equivalent to generation
       of Active Power
-    unit: MWh
+    unit:
+    - GWh
+
 - Active Power|Electricity|Electricity Storage:
     description: Active Power of energy storage systems, excl. hydro storage
-    unit: MWh
+    unit:
+    - GWh
+
+- Spillage|Electricity|Energy Storage System:
+    description: Energy spillage that could not be stored by an energy storage system
+      (ESS). Several energy models include spillage variables related to renewable
+      energy production; however, another way is to include a spillage variable to
+      the ESS unit since it is the last sink of the power network.
+    unit:
+    - GWh
+
+- Stored Energy Inventory|Electricity|Energy Storage System:
```

```suggestion
- Storage|Electricity|Energy Storage System:
```
to be consistant with the other technologies with storage